src/server/lib/choiceCards/junePromoChoiceCardSettings.ts

```diff
@@ -0,0 +1,105 @@
+import type { CountryGroupId } from '../../../shared/lib';
```

Comment:
This file is very similar to defaultChoiceCardSettings.ts, but with annual rate plans + copy